client/my-sites/stats/stats-list/stats-list-card.jsx

```diff
@@ -41,19 +32,19 @@ const StatsListPosts = ( {
 			emptyMessage={ emptyMessage }
 			isEmpty={ ! loader && ( ! data || ! data?.length ) }
 			titleURL={ titleURL }
-			className="list-posts-pages"
+			className={ `list-${ moduleType }-pages` }
 		>
 			{ !! loader && loader }
 			<HorizontalBarList data={ data }>
-				{ data?.map( ( item ) => {
+				{ data?.map( ( item, index ) => {
 					return (
 						<HorizontalBarListItem
-							key={ item?.id }
+							key={ item?.id || index } // not every item has an id
```

Comment:
Is there any chance of a collision between `id` and `index`?

Reply:
I will look into that but I think there shouldn't be conflicts because either the whole data set had ids or they don't.